src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -490,7 +493,15 @@ static AnnotationManager ofAllMinusExcluded(final List<String> annotationsToExcl
         }
 
         private static List<InfoFieldAnnotation> makeAllInfoFieldAnnotations() {
-            return ClassUtils.makeInstancesOfSubclasses(InfoFieldAnnotation.class, Annotation.class.getPackage());
+            // Get our configuration:
+            final GATKConfig config = ConfigFactory.getInstance().getGATKConfig();
+
+            final ClassFinder finder = new ClassFinder();
+            for ( final String codecPackage : config.annotation_packages() ) {
```

Comment:
Why do this instead of calling `ClassUtils.makeInstanceOfSubclasses` on each package and merging the results?

Reply:
This code is all going to be removed soon by https://github.com/broadinstitute/gatk/pull/4674. It might be best to wait for that, since it will reduce what this PR needs to do.

Reply:
How close is that PR to be accepted? I can divide this one into the `ReadFilter` config entry and wait for the annotator to be accepted to include the entry (or include it directly in #4674). What do you prefer?

Reply:
I'd prefer not to add anything else to #4674 (and also to not split this one up). I would just wait until its merged (@droazen still needs to review it) and then rebase this on top of that.